tests/functional/text_wer.py

```diff
@@ -0,0 +1,11 @@
+import pytest
```

Comment:
could you move these test to same file as the modular tests?
For testing we usually test both the functional and modular implementation in the same file because the tests are usually the same

Reply:
Done